src/test/java/com/mentorship/userservice/repository/AuthRepositoryTest.java

```diff
@@ -0,0 +1,96 @@
+package com.mentorship.userservice.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.mentorship.userservice.domain.Auth;
+import com.mentorship.userservice.domain.User;
+import com.mentorship.userservice.dto.enums.UserRole;
+import com.mentorship.userservice.repositories.UserRepository;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
+public class AuthRepositoryTest {
```

Comment:
JFYI: u can run db tests not only for h2 db, Look through the article:
https://www.baeldung.com/spring-boot-testcontainers-integration-test

Reply:
will be updated in the scope of another PR